spec.md

```diff
@@ -531,23 +531,51 @@ Credentials are a set of information used by an application or a user to utilize
+Brokers MUST NOT include a `syslog_drain_url` in a create binding response
+if it did not include a `"requires":["syslog_drain"]` property in its
+[Catalog](#catalog-management) endpoint.
 
 #### Route Services
 
-There are a class of service offerings that intermediate requests to applications, performing functions such as rate limiting or authorization. To configure a service instance with behavior specific to an application's routable address, a broker client MAY send the address along with the request to create a binding using `"bind_resource":{"route":"some-address.com"}`.
+There are a class of service offerings that intermediate requests to
+applications, performing functions such as rate limiting or authorization.
+If the platform and service both support this feature then the following
+applies:
 
-Some platforms MAY support proxying of application requests to service instances. In this case the platform needs to know where to send application requests; to facilitate this, the broker MUST return a `route_service_url` in the response for a request to create a binding. Not all services of this type expect to receive requests proxied by the platform; some services will have been configured out-of-band to intermediate requests to applications. In this case, the broker will not return `route_service_url` in response to the create binding request. By sending `bind_resource` as described above, the platform enables dynamic configuration of a service instance already in the application request path for the route, requiring no change in the platform routing tier.
+- To configure a service instance with behavior specific to an application's
+  routable address, a platform MUST send the address along with the request
+  to create a binding using `"bind_resource":{"route":"some-address.com"}`.
 
-The `requires` field in the [Catalog](#catalog-management) endpoint enables a platform marketplace to validate requests to create bindings. A platform MAY reject requests to create bindings when a broker has declared `"requires":["route_forwarding"]` for a service in the catalog endpoint.
+- Upon receiving an incoming application request, the platform needs to know
+  where to send this request to. To facilitate this, the broker MUST return
+  a `route_service_url` in the response for a request to create a binding.
+  However, if out-of-band configuration was performed then brokers MUST NOT
+  return return a `route_service_url` parameter.
+
+  By sending `bind-resource` as described above, the platform enables dynamic
+  configuration of a service instance already in the application request path
+  for the route, requiring no change in the platform routing tier.
+
+Brokers MUST NOT include a `route_service_url` in a create binding response
```

Comment:
I think the value of 'it' is ambiguous here, consider changing so it's explicit as the service the instance was provisioned from.